README.md

```diff
@@ -3,14 +3,118 @@
 [![Build Status](https://dev.azure.com/cnlabs/porter/_apis/build/status/deislabs.porter?branchName=master)](https://dev.azure.com/cnlabs/porter/_build/latest?definitionId=6?branchName=master)
 
 
-# Porter is a Cloud Installer
+# Porter is a tool for building cloud installers
```

Comment:
Can you revert this change for now? I don't want to change how we are branding/explaining Porter in this PR.